services/api/db/serializers.py

```diff
@@ -91,12 +94,14 @@ def validate(self, attrs):
         try:
             uid = urlsafe_base64_encode(force_bytes(user.id))
             token = PasswordResetTokenGenerator().make_token(user)
-        except Exception:
+            reset_link = reverse('reset-password-through-mail', kwargs={'uid': uid, 'token': token})
+        except ValidationError:
```

Comment:
```suggestion
        except ValidationError:
```

is abbove error class able to handle all possible error for uid, token and reverse